### PR TITLE
Add checks for passing explicit t-gradient to stiff solver

### PR DESCRIPTION
**Pull request by ScottPJones**

### CODE REVIEW

File: src/common.jl [hunk 1]
```diff
@@ -17,15 +17,21 @@ export daskr
...
-    timeseries_errors = true, save_everystep = isempty(saveat),
+    timeseries_errors = true, save_everystep = isempty(saveat), dense = save_everystep,
     save_start = true, save_timeseries = nothing,
     userdata = nothing, kwargs...)
 
-    verbose && !isempty(kwargs) && check_keywords(alg, kwargs, warnlist)
+    if verbose
+        !isempty(kwargs) && check_keywords(alg, kwargs, warnlist)
+        if !(typeof(prob.f) <: AbstractParameterizedFunction) && has_tgrad(prob.f)
+            warn("Explicit t-gradient given to this stiff solver is ignored. See http://docs.juliadiffeq.org/latest/basics/compatibility_chart.html")
```
Comment (ChrisRackauckas) [inline src/common.jl]: where's the `has_jac` version?

Comment (ScottPJones) [inline src/common.jl]: duh! forgot that!

Comment (ScottPJones) [inline src/common.jl]: done!

**Comment by ChrisRackauckas** [comment]
Waiting on DiffEqBase tag

**Pull request merged**